engine/engine.go

```diff
@@ -266,10 +268,14 @@ func PrintSettings(s *Settings) {
 	log.Debugf(log.Global, "\t Exchange HTTP timeout: %v", s.ExchangeHTTPTimeout)
 	log.Debugf(log.Global, "\t Exchange HTTP user agent: %v", s.ExchangeHTTPUserAgent)
 	log.Debugf(log.Global, "\t Exchange HTTP proxy: %v\n", s.ExchangeHTTPProxy)
+	log.Debugf(log.Global, "- GCTSCRIPT SETTINGS: ")
+	log.Debugf(log.Global, "\t Enable GCTScript manager: %v", s.EnableGCTScriptManager)
+	log.Debugf(log.Global, "\t GCTScript max virtual machines: %v", s.MaxVirtualMachines)
```

Comment:
This currently isn't linked, you can use the flagset to see if a user has toggled it and override the config value, otherwise use the config value

Reply:
:D whoops nice pick up the command line flag now properly gets passed to gctscriptconfig enabled field

Reply:
The many benefits of code review 😆

Reply:
s.MaxVirtualMachines will also need to be linked up, we assign the value on line 120 but we don't do anything with it after that

Reply:
Yep is coming in a push today just making some changes to mutexs as well